stock_dropshipping_dual_invoice/tests/test_need_two_invoices.py

```diff
@@ -73,14 +73,30 @@ def test_so_on_delivery_need_one_invoice(self):
         }).create({})
         self.assertIs(False, wizard._need_two_invoices())
 
+    def test_so_mto_need_one_invoice(self):
+        self.so_mto.order_policy = 'picking'
+        self.so_mto.action_button_confirm()
+        procurements = self.so_mto.procurement_group_id.procurement_ids
+        procurements.run()
+        po = procurements.filtered('purchase_id')[:1].purchase_id
+        self.assertTrue(po)
+        po.invoice_method = 'picking'
+        po.signal_workflow('purchase_confirm')
+        picking = po.picking_ids[:1]
```

Comment:
I know `[:1]` will accept an empty recordset: In fact I'd like to make that more explicit and remove hidden edge cases.

Thanks for your comment: I didn't realize we actually have two pickings. In that case, I'd love to have an explicit assertion that they are two, and then take one.

More importantly, if there are two and we want one, I guess we want to choose the right one, not just pick the first. What do you think? That might apply to the production code as well.